src/ancillary/io-ts-extra.ts

```diff
@@ -0,0 +1,96 @@
+import * as t from 'io-ts'
+import { CommitID, StreamID } from '@ceramicnetwork/streamid'
+import * as uint8arrays from 'uint8arrays'
+import { isDIDString } from './did-string.js'
+
+/**
+ * io-ts codec for JS `Uint8Array`.
+ */
+export const uint8array = new t.Type<Uint8Array, Uint8Array, unknown>(
+  'Uint8Array',
+  (input: unknown): input is Uint8Array => input instanceof Uint8Array,
+  function (this: t.Type<Uint8Array, Uint8Array, unknown>, input: unknown, context: t.Context) {
```

Comment:
Out of curiosity: why now use an arrow function expression like in other places :) ?

Reply:
An arrow function is bound to an outside `this`. Here though I need to use a different `this` and I want to command that to the TS compiler.